packages/esm-patient-list-app/src/patient-list-list/patient-list-list.component.tsx

```diff
@@ -81,6 +87,13 @@ const PatientListList: React.FC = () => {
         { id: 4, key: 'isStarred', header: '' },
       ];
 
+  const setSearchTerm = (searchString: string) => {
```

Comment:
Please rename this to `handleSearch`, since `setSearchTerm` and `setSearchTerms` are very much similar and hence confusing.